src/include/miopen/solver.hpp

```diff
@@ -1415,6 +1419,7 @@ struct ConvBinWinogradRxSf2x3 : SolverBase<ConvolutionContext>
     PerformanceConfigConvBinWinogradRxSf2x3 Search(const ConvolutionContext&) const;
 
     bool IsApplicable(const ConvolutionContext& params) const;
+    bool IsEnabledForNtHybrid() const { return true; }
```

Comment:
Why do we need to add this method to the solvers? Why not just get the tunable parameters from the performance config?

Reply:
We can find out if the performance config has parameters by adding either a `GetParameters` or `HasParameters` method to the `Serializable` class:

```cpp
    std::vector<std::string> GetParameters() const
    {
        std::vector<std::string> result;
        auto out = static_cast<const Derived&>(*this);
        Derived::Visit(out, [&](const std::string& name, auto&&) {
            result.push_back(name);
        });
        return result;
    }

    bool HasParameters() const
    {
        bool result = false;
        auto out = static_cast<const Derived&>(*this);
        Derived::Visit(out, [&](auto&&, auto&&) {
            result = true;
        });
        return result;
    }
```

Reply:
> We can find out if the performance config has parameters...

Thanks. However we do not need this. All performance configs have parameters. Otherwise we solver doesn't need (and doesn't "have") a PerformanceConfig class, and, therefore is does not have the GetSolution that takes two parameters:
```C++
    /// Searchable solvers provide a GetSolution that takes a Context and PerformanceConfig
    /// ConvSolution GetSolution(const ConvolutionContext& params,
    ///                          const PerformanceConfig& config) const;
```
This is enough for now.

Reply:
> Why do we need to add this method to the solvers? Why not just get the tunable parameters from the performance config?

Well, this is something different. 

During testing of the initial implementation, I've found that some untuned Solvers actually have good performance with default configs, so those should not be disabled to avoid perf drop (see https://github.com/ROCmSoftwarePlatform/MIOpen/pull/396#issuecomment-680135243). This method is very simple way to workaround this. It allows some tunable Solvers to participate in Find(), even if there is perf-db miss.

I do not like this -- basically this looks like a patch. However the whole PR looks like a crutch, so who cares. What I would really like to do is implement #410 and then get rid of both FAST_HYBRID and this NT_HYBRID.